starter-code/src/clue.js

```diff
@@ -1,13 +1,95 @@
 // ITERATION 1
 
 // Suspects Collection
-const suspectsArray = [];
+const suspectsArray = [
+    {
+     firstName: 'Jacob',
+     lastName: 'Green',
+     occupation: 'Entrepreneur',
+     age: 45,
+     description: 'He has a lot of connections',
+     image: mrGreenImage.src='//pbs.twimg.com/profile_images/506787499331428352/65jTv2uC.jpeg',
```

Comment:
this throws an error "mrgreenImage is not defined". I think you might have confused the image property of this particular object with an HTML tag. It was supposed to be:

image: '//pbs.twimg.com/profile_images/506787499331428352/65jTv2uC.jpeg',

Reply:
that was a mistake ... I wasn't that focused :)